litert/vendors/qualcomm/dispatch/litert_dispatch_invocation_context.cc

```diff
@@ -33,16 +36,34 @@
 #include "litert/core/util/tensor_type_util.h"
 #include "litert/vendors/c/litert_dispatch.h"
 #include "litert/vendors/qualcomm/context_binary_info.h"
+#include "litert/vendors/qualcomm/core/common.h"
 #include "litert/vendors/qualcomm/core/utils/miscs.h"
 #include "litert/vendors/qualcomm/dispatch/litert_dispatch_device_context.h"
 #include "litert/vendors/qualcomm/qnn_manager.h"
+#include "third_party/qairt/latest/include/QNN/HTP/QnnHtpProfile.h"
 #include "third_party/qairt/latest/include/QNN/QnnCommon.h"
 #include "third_party/qairt/latest/include/QNN/QnnTypes.h"
 
 using litert::Expected;
 using litert::Unexpected;
 using litert::qnn::QnnManager;
 
+std::string inline GetEventUnit(QnnProfile_EventUnit_t unit) {
```

Comment:
This will probably not allocate thanks to SSO. Nevertheless, could you use a string view? 
```suggestion
absl::string_view inline GetEventUnit(QnnProfile_EventUnit_t unit) {
```